docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -1275,8 +1275,8 @@ Contains Java Virtual Machine (JVM) statistics for the node.
 Last time JVM statistics were refreshed.
 
 `uptime`::
-(<<time-units,time value>>)
-JVM uptime.
+(optional, <<time-units,time value>>)
+Human readable JVM uptime. Enabled by the `human` parameter.
```

Comment:
```suggestion
Human-readable JVM uptime. Enabled by the `human` parameter.
```

Reply:
hah I originally had a hyphen and thought it might be better the other way, but I was probably right in the first draft. Accepting suggestion.